src/features/targetedOutreach/hooks/useShowOutreachPopup.tsx

```diff
@@ -0,0 +1,26 @@
+import useIsSafeOwner from '@/hooks/useIsSafeOwner'
+import { MAX_ASK_AGAIN_DELAY } from '@/features/targetedOutreach/constants'
+import { useAppSelector } from '@/store'
+import { selectCookieBanner } from '@/store/popupSlice'
+import type { getSubmission } from '@safe-global/safe-client-gateway-sdk'
+
+export const useShowOutreachPopup = (
+  isDismissed: boolean | undefined,
+  askAgainLaterTimestamp: number | undefined,
+  submission: Awaited<ReturnType<typeof getSubmission>> | undefined,
```

Comment:
```suggestion
  submission: getSubmission | undefined,
```

There is a type export named identical to the function export but it should work to use it as a type.